src/cares_wrap.cc

```diff
@@ -1266,8 +1266,14 @@ static void Initialize(Handle<Object> target,
               Integer::New(env->isolate(), AF_UNSPEC));
   target->Set(FIXED_ONE_BYTE_STRING(env->isolate(), "AI_ADDRCONFIG"),
               Integer::New(env->isolate(), AI_ADDRCONFIG));
+#ifdef __FreeBSD__
```

Comment:
I remember that it was broken in FreeBSD 8 but I would be sad and surprised if it's still broken in FreeBSD 10.  Could it be that the tests fail because we allow V4MAPPED with IPv4 queries?  Does the patch below fix the test?

```diff
diff --git a/lib/dns.js b/lib/dns.js
index 0a6e84a..74af0b2 100644
--- a/lib/dns.js
+++ b/lib/dns.js
@@ -113,6 +113,9 @@ exports.lookup = function lookup(hostname, options, callback) {
   if (family !== 0 && family !== 4 && family !== 6)
     throw new TypeError('invalid argument: family must be 4 or 6');
 
+  if (family !== 6 && (hints & exports.V4MAPPED))
+    throw new TypeError('V4MAPPED only supported for IPv6 queries');
+
   callback = makeAsync(callback);
 
   if (!hostname) {
diff --git a/test/parallel/test-dns.js b/test/parallel/test-dns.js
index 1de9bbf..6da04be 100644
--- a/test/parallel/test-dns.js
+++ b/test/parallel/test-dns.js
@@ -135,12 +135,14 @@ assert.doesNotThrow(function() {
 
 assert.doesNotThrow(function() {
   dns.lookup('www.google.com', {
+    family: 6,
     hints: dns.V4MAPPED
   }, noop);
 });
 
 assert.doesNotThrow(function() {
   dns.lookup('www.google.com', {
+    family: 6,
     hints: dns.ADDRCONFIG | dns.V4MAPPED
   }, noop);
 });
```